internal/cli/atlas/security/customercerts/disable.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package customercerts
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DisableOpts struct {
+	cli.GlobalOpts
+	store   store.X509CertificateDisabler
+	confirm bool
+}
+
+func (opts *DisableOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *DisableOpts) Run() error {
+	if !opts.confirm {
+		fmt.Printf("X.509 configuration was not disabled.\n")
+		return nil
+	}
+
+	if err := opts.store.DisableX509Configuration(opts.ConfigProjectID()); err != nil {
+		return err
+	}
+
+	fmt.Printf("X.509 configuration for project %s was deleted.\n", opts.ConfigProjectID())
+
+	return nil
+}
+
+func (opts *DisableOpts) Prompt() error {
+	prompt := &survey.Confirm{
+		Message: "Are you sure you want to delete the X509 configuration for this project?",
```

Comment:
Thought of using the DeleteOpts but the language is slightly off if there is no specific resource being deleted. So I just implemented the questions here rather than modify delete opts